libraries-bom/pom.xml

```diff
@@ -4,7 +4,7 @@
   <!-- adding comment for testing-->
   <groupId>com.google.cloud</groupId>
   <artifactId>libraries-bom</artifactId>
-  <version>26.2.1-SNAPSHOT</version><!-- {x-version-update:libraries-bom:current} -->
+  <version>26.2.1</version><!-- {x-version-update:libraries-bom:current} -->
```

Comment:
@ddixit14 Do you know why we're not getting the minor version bump here despite `google-cloud-bom` which is a dependency of `libraries-bom` having a minor version bump? Do we need some additional configuration in https://github.com/googleapis/java-cloud-bom/blob/main/release-please-config.json?